plugins/module_utils/opnsense_utils.py

```diff
@@ -35,7 +35,11 @@ def run_function(
     params_string = ",".join(configure_params)
 
     # assemble php command
-    php_cmd = f"{requirements_string} {configure_function}({params_string});"
+    if params_string:
+        php_cmd = f"{requirements_string} {configure_function}({params_string});"
+    else:
+        php_cmd = f"{requirements_string} {configure_function};"
```

Comment:
Shouldn't the configure_fucntion be called with `()`?
```suggestion
        php_cmd = f"{requirements_string} {configure_function}();"
```

Reply:
I would rather not implement this, since it  works without brackets as expected.